pages/Documents/Security&Authentication/AppleBusinessChatAuthentication/overview.md

```diff
@@ -3,11 +3,11 @@ pagename: Overview
 Keywords:
 sitesection: Documents
 categoryname: "Security & Authentication"
-documentname: Apple Business Chat Authentication
-permalink: apple-business-chat-authentication-overview.html
+documentname: Apple Messages for Business Authentication
```

Comment:
`documentname` needs updating in documentsupdated.yaml.

Also, please add a redirect.

Reply:
reverted -- will address page names and redirects separately